app/common/src/image/StretchedImage.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import * as React from 'react';
+import { Image, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  image: {
+    ...StyleSheet.absoluteFillObject,
+    width: null,
+    height: null,
+  },
+});
+
+export default function StretchedImage({ source }: { source: any }) {
+  return <Image source={source} resizeMode="stretch" style={styles.image} />;
```

Comment:
Maybe just `<Image {...props} resizeMode="stretch" />`? Your implementation may be limitting in the future if you want to use additional props.